sal_interfaces/Scheduler/Scheduler_Events.xml

```diff
@@ -368,4 +402,576 @@
         <Units>unitless</Units>
         <Count>1</Count>
     </item>
-</SALEvent></SALEventSet>
+</SALEvent>
+<SALEvent>
+    <Subsystem>Scheduler</Subsystem>
+    <EFDB_Topic>Scheduler_logevent_schedulerConfig</EFDB_Topic>
+		<Description>Base Scheduler configuration.</Description>
+    <item>
+        <EFDB_Name>surveyDuration</EFDB_Name>
+        <Description>Survey duration.</Description>
+        <IDL_Type>double</IDL_Type>
+        <Units>year</Units>
+        <Count>1</Count>
+    </item>
+</SALEvent>
+<SALEvent>
+    <Subsystem>Scheduler</Subsystem>
+    <EFDB_Topic>Scheduler_logevent_driverConfig</EFDB_Topic>
+		<Description>Basic Scheduler driver configuration.</Description>
+    <item>
+        <EFDB_Name>nightBoundary</EFDB_Name>
+        <Description>Sun altitude that marks the beginning/end of the night.</Description>
+        <IDL_Type>double</IDL_Type>
+        <Units>deg</Units>
+        <Count>1</Count>
+    </item>
+    <item>
+        <EFDB_Name>newMoonPhaseThreshold</EFDB_Name>
+        <Description>Moon illumination that marks the beginning/end of dark nights</Description>
+        <IDL_Type>double</IDL_Type>
+        <Units>unitless</Units>
+        <Count>1</Count>
+    </item>
+    <item>
+        <EFDB_Name>startupType</EFDB_Name>
+        <Description>What type of startup should the scheduler do? COLD, WARM or HOT?</Description>
+        <IDL_Type>string</IDL_Type>
+        <IDL_Size>1</IDL_Size>
+        <Units>unitless</Units>
+        <Count>1</Count>
+    </item>
+    <item>
+        <EFDB_Name>startupDatabase</EFDB_Name>
+        <Description>If WARM or HOT start, give the path to the startup database.</Description>
+        <IDL_Type>string</IDL_Type>
+        <IDL_Size>1</IDL_Size>
+        <Units>unitless</Units>
+        <Count>1</Count>
+    </item>
+</SALEvent>
+<SALEvent>
+    <Subsystem>Scheduler</Subsystem>
+    <EFDB_Topic>Scheduler_logevent_obsSiteConfig</EFDB_Topic>
+		<Description>Observatory model: site configuration.</Description>
+    <item>
+        <EFDB_Name>name</EFDB_Name>
```

Comment:
This is a reserved keyword and the unit tests choke on it. Please rename to e.g. observatory_name.